io/net/infiniband/ip_over_ib.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Prudhvi Miryala<mprudhvi@linux.vnet.ibm.com>
+# Ip Over IB test
+
+
+import os
+import time
+import netifaces
+from avocado import Test
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils import build
+from avocado.utils import git
+from avocado.utils import process
+from avocado.utils import distro
+
+
+class ip_over_ib(Test):
+    '''
+    Ip Over IB Test
+    '''
+    def setUp(self):
+        '''
+        To check and install dependencies for the test
+        '''
+        sm = SoftwareManager()
+        detected_distro = distro.detect()
+        depends = ["openssh-clients", "git"]
+        for package in depends:
+            if not sm.check_installed(package):
+                self.skip("%s package is need to test" % package)
+        interfaces = netifaces.interfaces()
+        self.IF = self.params.get("Iface", default="")
+        self.PEER_IP = self.params.get("PEERIP", default="")
+        if self.IF not in interfaces:
+            self.skip("%s interface is not available" % self.IF)
+        if self.PEER_IP == "":
+            self.skip("%s peer machine is not available" % self.PEER_IP)
+        self.to = self.params.get("timeout", default="120")
+        self.IPERF_RUN = self.params.get("IPERF_RUN", default="0")
+        self.NETSERVER_RUN = self.params.get("NETSERVER_RUN", default="0")
+        # source: git clone git://git.linux.ibm.com/power-io-fvt/eio_src.git
+        git.get_repo('git://git.linux.ibm.com/power-io-fvt/e\
+                     io_src.git', destination_dir=self.srcdir)
```

Comment:
this git repo not able to access from outside IBM network can you please use url where it hosted in public network